README.md

```diff
@@ -30,52 +28,115 @@ the branch tracking the latest 0.7 release.
+platform to securely manage remote access for any-sized organization. Unlike
+most VPNs, Firezone takes a granular, least-privileged approach to access
+management with group-based policies that control access to individual
+applications, entire subnets, and everything in between.
 
+<p align="center">
+  <img width="1439" alt="architecture" src="https://github.com/firezone/firezone/assets/167144/48cd6a1e-2f3f-4ca7-969a-fc5b33e13d1c">
+</p>
+
+<!-- TODO: New intro usage video
 ![firezone-usage](https://user-images.githubusercontent.com/52545545/147392573-fe4cb936-a0a8-436f-a69b-c0a9587de58b.gif)
+ -->
 
-- **Fast:** Uses WireGuard® to be
-  [3-4 times](https://wireguard.com/performance/) faster than OpenVPN.
-- **SSO Integration:** Authenticate using any identity provider with an OpenID
-  Connect (OIDC) connector.
-- **Containerized:** All dependencies are bundled via Docker.
-- **Simple:** Takes minutes to set up. Manage via a simple CLI.
-- **Secure:** Runs unprivileged. HTTPS enforced. Encrypted cookies.
-- **Firewall included:** Uses Linux [nftables](https://netfilter.org) to block
-  unwanted egress traffic.
+## Features
 
-### Anti-features
+Firezone is:
+
+- **Fast:** Built on WireGuard® to be
+  [3-4 times](https://wireguard.com/performance/) faster than OpenVPN.
+- **Scalable:** Deploy two or more gateways for automatic load balancing and
+  failover.
+- **Private:** Tunnels are peer-to-peer so packets never route through Firezone
+  infrastructure.
+- **Secure:** Zero attack surface thanks to Firezone's holepunching tech which
+  establishes tunnels on-the-fly at the time of access.
+- **Flexible:** Authenticate users via email, Google Workspace, or OIDC and sync
+  users and groups automatically.
+- **Simple:** Deploy gateways and configure access in minutes with a snappy
+  admin UI.
 
 Firezone is **not:**
 
-- An inbound firewall
-- A tool for creating mesh networks
-- A full-featured router
+- A tool for creating bi-directional mesh networks
+- A full-featured router or firewall
 - An IPSec or OpenVPN server
 
+## Contents of this repository
+
+This is a monorepo containing the full Firezone product, marketing website, and
+product documentation, organized as follows:
+
+- [elixir/](./elixir): Control plane API and admin UI
+- [rust/](./rust): Data plane components and Linux / Windows clients
+- [swift/](./swift/apple): macOS / iOS clients
+- [kotlin/](./kotlin/android): Android / ChromeOS clients
+- [website/](./website): Marketing website and product documentation
+
+## Frequently asked questions (FAQ)
+
+> **Can I self-host Firezone?**
+
+Our [license](#license) won't stop you from self-hosting the entire Firezone
+product top to bottom, but we can't dedicated the resources to make this a
+smooth experience and therefore don't support self-hosting the control plane at
+this time.
+
+If you have a business case requiring an onprem installation of Firezone please
```

Comment:
Minor grammatical nit...I think `onprem` either needs to have a space or a hyphen.

Reply:
👍 Updated